components/Accordion.jsx

```diff
@@ -26,7 +26,7 @@ export default function Accordion() {
           <p className="w-[80%]">When and where does SCI take place?</p>
           <IoArrowDownCircleOutline size={30} color="white" className={`ease-in-out duration-300 ${active ? 'rotate-180' : 'rotate-0'}`} />
         </div>
-        <div className={`bg-white rounded-b-md overflow-hidden ease-in-out duration-300 ${active ? 'max-h-[500px]' : 'max-h-0'}`}>
+        <div className={`bg-white bg-opacity-25 text-white bg-opacity-10 text-white rounded-b-md overflow-hidden ease-in-out duration-300 ${active ? 'max-h-[500px]' : 'max-h-0'}`}>
```

Comment:
You have two `bg-opacity` and two `tex-white` attributes, please remove one of each 🙏🏻

Reply:
Hello Daniel, thanks for your review. 

My friend suffers from a diagnosed Lack of Attention so it is hard for him to pay attention to those small details. 
We'll fix that soon 🙏🏻